README.md

```diff
@@ -52,18 +52,59 @@ To run the full suite, run: `npm test`.
+
+## Privacy
+
+At New Relic we take your privacy and the security of your information seriously, and are
+committed to protecting your information. We must emphasize the importance of not sharing
+personal data in public forums, and ask all users to scrub logs and diagnostic information
+for sensitive information, whether personal, proprietary, or otherwise.
+
+We define "Personal Data" as any information relating to an identified or identifiable
+individual, including, for example, your name, phone number, post code or zip code,
+Device ID, IP address and email address.
+
+Please review [New Relic’s General Data Privacy Notice](https://newrelic.com/termsandconditions/privacy) for more information.
+
+## Roadmap
+See our [roadmap](https://github.com/newrelic/node-newrelic/blob/main/ROADMAP_Node.md), to learn more about our product vision,
+understand our plans, and provide us valuable feedback. Remove this section if it’s not needed.
 
 ## Contributing
-We encourage your contributions to improve New Relic MySQL instrumentation!  Keep in mind when you
-submit your pull request, you'll need to sign the CLA via the click-through using CLA-Assistant. You
-only have to sign the CLA one time per project.
+We encourage your contributions to improve New Relic MySQL instrumentation!  Keep in mind
+when you submit your pull request, you'll need to sign the CLA via the click-through using
+CLA-Assistant. You only have to sign the CLA one time per project.
+If you have any questions, or to execute our corporate CLA, required if your contribution
+is on behalf of a company,  please drop us an email at opensource@newrelic.com.
+
+**A note about vulnerabilities**
+
+As noted in our [security policy](https://github.com/newrelic/.github/blob/master/SECURITY.md), New Relic is committed to
+the privacy and security of our customers and their data. We believe that providing
+coordinated disclosure by security researchers and engaging with the security community
+are important means to achieve our security goals.
+
+If you believe you have found a security vulnerability in this project or any of New Relic's
+products or websites, we welcome and greatly appreciate you reporting it to New Relic
+through [HackerOne](https://hackerone.com/newrelic).
+
+If you would like to contribute to this project, please review [these guidelines](https://github.com/newrelic/node-newrelic-mysql/blob/main/CONTRIBUTING.md).
+
+To [all contributors](https://github.com/newrelic/node-newrelic-mysql/graphs/contributors),we thank you!  Without your contribution,
+this project would not be what it is today.
 
-If you have any questions, or to execute our corporate CLA, required if your contribution is on
-behalf of a company,  please drop us an email at opensource@newrelic.com.
+We also host a community project page dedicated to
+[the New Relic MySql Instrumentation package](https://opensource.newrelic.com/newrelic/node-newrelic-mysql).
```

Comment:
Just calling out this doesn't exist yet but should once we wrap up work (assuming that is the right link, not sure)